RequestManagementSystem/ConfigTemplate.cfg

```diff
@@ -10,17 +10,17 @@ Services
   }
   ReqProxy
   {
-    Port = 9198
+    Port = 9161
```

Comment:
Hi, I requested the change to 9161, as that is the port used by LHCb (and I thought everybody) for this. New installations were failing and each installation needed this service to be deleted and re-installed by hand due to this bug.

What is the reason this was reverted?

Thanks,
Raja.